octue/cli.py

```diff
@@ -341,6 +370,42 @@ def dataflow(service_config, no_cache, update, dataflow_job_only, image_uri):
     deployer.deploy(no_cache=no_cache, update=update)
 
 
+@deploy.command()
+@click.argument("project_name")
+@click.argument("service_namespace")
+@click.argument("service_name")
+@click.argument("push_endpoint")
+@click.option(
+    "--revision-tag",
+    is_flag=False,
+    default=None,
+    show_default=True,
+    help="The service revision tag (e.g. 1.0.7). If this option isn't given, a random 'cool name' tag is generated e.g"
+    ". 'curious-capybara'.",
+)
+def create_push_subscription(project_name, service_namespace, service_name, push_endpoint, revision_tag):
+    """Create a push subscription on Google Pub/Sub from the Octue service to the push endpoint. If a corresponding
+    topic doesn't exist, it will be created.
+
+    PROJECT_NAME is the name of the Google Cloud project in which the subscription will be created
+
+    SERVICE_NAMESPACE is the namespace the service belongs to in kebab case
+
+    SERVICE_NAME is the name of the service in kebab case, unique within its namespace
+
+    PUSH_ENDPOINT is the HTTP/HTTPS endpoint of the service to push to. It should be fully formed and include the
+    'https://' prefix
+    """
+    service_sruid = create_service_sruid(namespace=service_namespace, name=service_name, revision_tag=revision_tag)
+    pub_sub_sruid = convert_service_id_to_pub_sub_form(service_sruid)
+
+    topic = Topic(name=pub_sub_sruid, project_name=project_name)
+    topic.create(allow_existing=True)
+
+    subscription = Subscription(name=pub_sub_sruid, topic=topic, project_name=project_name, push_endpoint=push_endpoint)
+    subscription.create()
+
+
 def _add_monitor_message_to_file(path, monitor_message):
```

Comment:
This should probably print out the revision tag (or perhaps the whole SRUID) that gets used, so the result is pipable to a variable in bash

Reply:
Which would you like?
1. The user-friendly SRUID `octue/example-service:0.1.2`
2. The Pub/Sub-friendly SRUID `octue.example-service.0-1-2`
3. The Pub/Sub-friendly SRUID + services namespace `octue.services.octue.example-service.0-1-2`
4. The Pub/Sub path `projects/my-project/subscriptions/octue.services.octue.example-service.3-5-0`

Reply:
I'll go with `3` for now